opendevin/server/session.py

```diff
@@ -38,7 +38,6 @@
 
 
 DEFAULT_WORKSPACE_DIR = os.getenv("WORKSPACE_DIR", os.path.join(os.getcwd(), "workspace"))
-LLM_MODEL = os.getenv("LLM_MODEL", "gpt-4-0125-preview")
```

Comment:
I think here is unnecessary because `opendevin/llm/llm.py` will also read default value so i propose translate the work of reading the default configuration to there.

Reply:
This is actually a good point--we should remove